Ginger/Ginger/UserControlsLib/UCElementDetails.xaml.cs

```diff
@@ -440,38 +440,6 @@ public void InitLegacyFriendlyLocatorsGridView()
             }
         }
 
-        internal void InitFriendlyLocatorsGridView()
```

Comment:
why this function was removed, did we replace it with other function?

Reply:
this function is for initiating friendly locator grid on windows explorer and live spy page but we don't need it over their that's why it's removed